gr-utils/python/modtool/core/makeyaml.py

```diff
@@ -147,13 +147,16 @@ def _type_translate(p_type, default_v=None):
                 return 'raw'
         def _get_blockdata(fname_cc):
             """ Return the block name and the header file name from the .cc file name """
+            flag = False
```

Comment:
Please give the variable a meaningful name which explains its existence at first glance.

Reply:
Sure!